source/intro_to/explore_software.rst

```diff
@@ -0,0 +1,229 @@
+
+:bash:`antsApplyTransforms` To apply any transformations from ANTs to other images. Useful after registration of two images to transform masks and labels. Be sure to apply the transformation in the right order and use the right datatype (ask Francois Rheault about it).
+
+:bash:`antsMultivariateTemplateConstruction2.sh` To build templates. Suppose you want to build a FA or T1 template of your aging brain population (or any population). We use ANTS for this. This script is fairly complex, you have to know a lot about the inner working of ANTs to obtain a good results (ask Francois Rheault about it)
+
+
+:bash:`antsBrainExtraction.sh` To perform skull-stripping or brain extraction. This is done with registration and a template. It usually takes 20-30 minutes but can be more robust than FSL bet (we use it only for T1w, FLAIR). The template is available `(here) <https://github.com/scilus/containers-tractoflow/raw/master/human-data_master_1d3abfb.tar.bz2>`__ (use the *t1_template.nii.gz* and *t1_brain_probability_map.nii.gz*)
+
+:bash:`N4BiasFieldCorrection` To perform bias correction algorithm on T1w, which evaluate the spatial change in signal and ensure its uniformity.
+
+:bash:`antsAtroposN4.sh` To segment WM-GM-CSF and to compute cortical thickness. This is a less common usage in the lab.
+
+
+Installation
+^^^^^^^^^^^^^^^^^^^^^
+
+Visit this website for download and install instructions `(here) <https://github.com/ANTsX/ANTs/wiki/Compiling-ANTs-on-Linux-and-Mac-OS>`__. ANTs use a pretty complex algorithm, so the command line is sometimes hard to understand at first, `this <https://github.com/ANTsX/ANTs/wiki/Anatomy-of-an-antsRegistration-call>`__. should help.
+
+.. code-block:: bash
+
+    export ANTSPATH="/PATH/TO/ANTS/ants/build/install/bin"
+    export PATH=${ANTSPATH}:${PATH}
+
+Once installed, you should be able to type :bash:`antsRegistration` to see the help display.
+
+MRtrix
+-----------------------
+MRtrix is an open-source software package for diffusion MRI analysis and Tractography. It provides a suite of tools for processing, analyzing, and visualizing diffusion MRI data and streamlines. It is widely used in research and clinical applications.
+Visit this `website <https://www.mrtrix.org/>`__ for download and install instructions as well as a general overview of tools available in `documentation <https://mrtrix.readthedocs.io/en/latest/>`__
+
+For a quick overview of commands you can look at their Getting Started `(here) <https://mrtrix.readthedocs.io/en/latest/getting_started/beginner_dwi_tutorial.html>`__ section. This `webpage <https://osf.io/fkyht/>`__  hosts an extensive tutorial of MRtrix3 from raw data to tractography (and more). We recommend trying this tutorial and finding analogous functions in Scilpy to reach the same end goals.
+
+MRtrix is another set of tools to analyse diffusion data, much like Dipy and Scilpy. One main difference is that they use C++ whereas we use python.
```

Comment:
, which may be slower but is easier to use open-source because of its readability?